.github/workflows/check-pl-compat.yaml

```diff
@@ -111,7 +111,6 @@ jobs:
         CXX_COMPILER=$(which g++ }}) \
         ENABLE_LLD=OFF \
         RT_BUILD_DIR="$(pwd)/runtime-build" \
-        LIGHTNING_GIT_TAG_VALUE=master \
```

Comment:
I don't think we need these three sections anymore, since they existed to select the appropriate lightning version

Reply:
In fact we should be able to move it up into the section above like so:

```
        CCACHE_DIR="$(pwd)/.ccache" \
        C_COMPILER=$(which gcc }}) \
        CXX_COMPILER=$(which g++ }}) \
        RT_BUILD_DIR="$(pwd)/runtime-build" \
        LLVM_BUILD_DIR="$(pwd)/llvm-build" \
        MHLO_BUILD_DIR="$(pwd)/mhlo-build" \
        ENZYME_BUILD_DIR="$(pwd)/enzyme-build" \
        DIALECTS_BUILD_DIR="$(pwd)/quantum-build" \
        ENABLE_LLD=ON \
        make catalyst
```

The `catalyst` target is a combination of `runtime`, `dialects`, `frontend`.

Reply:
Done